### PR TITLE
Release Notes for 6.4.0

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -16,6 +17,93 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-4-0]]
+=== Logstash 6.4.0 Release Notes
+
+* Adds the Azure Module.
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: This could probably do with a little embellishment

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 This section summarizes the changes in the following releases:
 
+* <<Logstash-6-4-0,Logstash 6.4.0>>
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: `<<Logstash` should be `logstash` (lowercase).  The link's not resolving and doc won't build.

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -16,6 +17,93 @@ This section summarizes the changes in the following releases:
...
+* Javafication to continue moving parts of Logstash core from Ruby to Java and some general code cleanup ({lsissue}9414[Issue 9414]), {lsissue}9415[Issue 9415], {lsissue}9416[Issue 9416], {lsissue}9422[Issue 9422]), {lsissue}9482[Issue 9482], {lsissue}9486[Issue 9486], {lsissue}9489[Issue 9489], {lsissue}9490[Issue 9490], {lsissue}9491[Issue 9491], {lsissue}9496[Issue 9496], {lsissue}9520[Issue 9520], {lsissue}9587[Issue 9587], {lsissue}9574[Issue 9574], {lsissue}9610[Issue 9610], {lsissue}9620[Issue 9620], {lsissue}9631[Issue 9631], {lsissue}9632[Issue 9632], {lsissue}9633[Issue 9633], {lsissue}9661[Issue 9661], {lsissue}9662[Issue 9662], {lsissue}9665[Issue 9665], {lsissue}9667[Issue 9667], {lsissue}9668[Issue 9668], {lsissue}9670[Issue 9670], {lsissue}9676[Issue 9676], {lsissue}9687[Issue 9687], {lsissue}9693[Issue 9693], {lsissue}9697[Issue 9697], {lsissue}9699[Issue 9699], {lsissue}9717[Issue 9717], {lsissue}9723[Issue 9723], {lsissue}9731[Issue 9731], {lsissue}9740[Issue 9740], {lsissue}9742[Issue 9742], {lsissue}9743[Issue 9743], {lsissue}9751[Issue 9751], {lsissue}9752[Issue 9752], {lsissue}9765[Issue 9765]).
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: Remove extraneous parens:
screen shot 2018-08-09 at 6 06 52 pm
I suggest leaving the first opening paren and the last closing paren, and deleting the others.

File: docs/static/releasenotes.asciidoc (hunk 2 above)
Comment (robbavey) [reply docs/static/releasenotes.asciidoc]: @karenzone oops - I've pushed a commit that fixes this

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -16,6 +17,93 @@ This section summarizes the changes in the following releases:
...
+[float]
+==== Plugins
+
+*Rubydebug Codec*
+
+* Fixes crash that could occur on startup if $HOME was unset or if ${HOME}/.aprc was unreadable by pinning awesome_print dependency to a release before the bug was introduced. https://github.com/logstash-plugins/logstash-codec-rubydebug/pull/5[#5]
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: $HOME and ${HOME}/.aprc need backticks.  Asciidoc won't render as it is.

**Comment by karenzone** [comment]
The first link to the azure plugin `{logstash-ref}/plugins-input-azure_event_hubs.html[Azure event hub input]` is missing the `s` on `inputs`.  
The second mention works correctly. `{logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs input plugin]` *Consider replacing the first instance with this once, since Alvin wants to emphasize that this is an input plugin.

**Comment by karenzone** [comment]
Builds cleanly. Links resolve.  LGTM.

**Comment by robbavey** [comment]
@karenzone I added notes for the changes made via Kibana - the config viewer changes we talked about, and also changes to the grok debugger.

**Pull request merged**